lib/array.js

```diff
@@ -14,7 +14,8 @@ function defaults (opts) {
     const assigning = !prop || !prop.getable
     const allowEmpty = rawSchema.allowEmpty !== false && allowEmptyArrays
 
-    const a = assigning ? gen.sym(rawSchema.name || 'arr') : prop.get()
```

Comment:
this is a bug in generate-function. .sym should always be safe to use. the name provided is just a hint to ease debugging. I'll fix those there.

Reply:
(fixed in master) all .sym calls are safe now with the latest generate-function